src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -51,6 +53,7 @@ module.exports = class LevelPlane extends Array {
           }
         }
       });
+      this.getRedstone();
```

Comment:
I think we also need to call this when placing `railsRedstoneTorch`?

Reply:
That is a great point... last time I heard we -weren't- placing torches? I don't think? Something was going to be static last time I spoke with the designers. I'll absolutely add the case because who cares if the designers don't use it, but I'll confirm it anyway